pyramid_oereb/standard/create_standard_models.py

```diff
@@ -9,6 +9,15 @@ def create_standard_model():
         description='Create a python file which contais all sqlalchemy models in to hold oereb '
                     'specification like data'
     )
+
+    def str2bool(v):
```

Comment:
never used ...

Reply:
=> will mean that's his usage is missing :-)

Reply:
Ok :)) I understand :P